.swiftlint.yml

```diff
@@ -42,6 +42,9 @@ variable_name:
     - rs
     - URI
     - URL
+type_name:
+  excluded:
+    - T
```

Comment:
There was a associatedtype definition that the latest version of swiftlint was complaining about.